src/metrics.ts

```diff
@@ -883,9 +869,9 @@ export function getMetrics (
     },
 
     onDuplicateMsgDelivery (topicStr: TopicStr, deliveryDelayMs: number, isLateDelivery: boolean): void {
-      this.duplicateMsgDeliveryDelay.observe(deliveryDelayMs / 1000)
+      const topic = this.toTopic(topicStr)
+      this.duplicateMsgDeliveryDelay.observe({ topic }, deliveryDelayMs / 1000)
```

Comment:
Label was defined in type but not set. I'd assume this was just an oversight as topic is always available here.